components/expression_language.rst

```diff
@@ -66,11 +66,11 @@ The main class of the component is
 
     use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
 
-    $language = new ExpressionLanguage();
+    $expressionLanguage = new ExpressionLanguage();
```

Comment:
I don't like either the original $language nor the new $expressionLanguage. Neither of them tells me what is this. But I guess the new name is better than the old one.

Reply:
At least we are calling an apple an apple here :)